### PR TITLE
Python scheduler docker image

### PR DESCRIPTION
**Pull request by fishinthecalculator**

### CODE REVIEW

**Comment by fishinthecalculator** [comment]
https://github.com/Tech-Workers-Coalition-Italia/mobilizon-reshare/runs/4973187705

**Comment by fishinthecalculator** [comment]
https://github.com/Tech-Workers-Coalition-Italia/mobilizon-reshare/actions/runs/1786466375

File: README.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ The goal of `mobilizon_reshare` is to provide a suite to reshare Mobilizon event
 tool enables an organization to automate their social media strategy in regards
 to events and their promotion. 
 
+# Platforms
+
+`mobilizon-reshare` currently supports the following social platforms:
+
+- Mastodon
```
Comment (chobeat) [inline README.md:11]: You missed facebook

**Review by chobeat — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (fishinthecalculator) [reply README.md:12]: Fixed thanks

**Pull request merged**